backend/fms_core/tests/test_services/experiment_run_test.py

```diff
@@ -0,0 +1,144 @@
+                                                                                   container_obj=self.container,
+                                                                                   instrument_obj=self.instrument,
+                                                                                   samples_info=self.samples_info,
+                                                                                   process_properties=self.properties,
+                                                                                   start_date=self.start_date
+                                                                                   )
+        self.assertEqual(my_experiment_run, None)
+        self.assertTrue('Run type is required to create an experiment run.' in errors)
+        self.assertEqual(warnings, [])
+
+    def test_missing_process_properties(self):
+        my_experiment_run, errors, warnings = experiment_run.create_experiment_run(experiment_run_name=self.experiment_name,
+                                                                                   run_type_obj=self.run_type,
+                                                                                   container_obj=self.container,
+                                                                                   instrument_obj=self.instrument,
+                                                                                   samples_info=self.samples_info,
+                                                                                   process_properties=None,
+                                                                                   start_date=self.start_date
+                                                                                   )
+        self.assertEqual(my_experiment_run, None)
+        self.assertTrue('Process properties are required to create an experiment run.' in errors)
+        self.assertEqual(warnings, [])
+
+    def test_missing_parameters(self):
+        my_experiment_run, errors, warnings = experiment_run.create_experiment_run(experiment_run_name=None,
+                                                                                   run_type_obj=self.run_type,
+                                                                                   container_obj=None,
+                                                                                   instrument_obj=None,
+                                                                                   samples_info=self.samples_info,
+                                                                                   process_properties=self.properties,
+                                                                                   start_date=self.start_date
+                                                                                   )
+
+        self.assertEqual(my_experiment_run, None)
+        self.assertTrue('This field cannot be null.' in errors[0])
```

Comment:
Mmm if this is really the error message there might be an upgrade here to do ... Knowing the name of the field would be great...